drivers/crypto/Kconfig.nrf_ecb

```diff
@@ -0,0 +1,18 @@
+# NRF AES ECB configuration options
+
+# Copyright (c) 2020 Nordic Semiconductor ASA
+# SPDX-License-Identifier: Apache-2.0
+
+
+menuconfig CRYPTO_NRF_ECB
+	bool "nRF AES electronic codebook mode encryption"
```

Comment:
This is probably incompatible with the BLE controller, so we probably need to add a `depends on !BT_CTLR` or similar. cc @cvinayak

Reply:
done